app/helpers/select_options_helper.rb

```diff
@@ -0,0 +1,19 @@
+module SelectOptionsHelper
```

Comment:
I can't see this included anywhere - is there some magic?

Reply:
I think Rails autoloads everything in `app/helpers`, but I may be mistaken!

Reply:
> I can't see this included anywhere - is there some magic?

Yes - Rails makes all helpers available in all views.